scroll-animations-1/Overview.bs

```diff
@@ -648,35 +653,72 @@ The procedure to calculate <dfn>effective scroll range</dfn> of a
 
     </div>
 
-### The current time of a {{ScrollTimeline}} ### {#current-time-algorithm}
 
-The [=current time=] of a {{ScrollTimeline}} is calculated as follows:
+### The phase of a {{ScrollTimeline}} ### {#phase-algorithm}
 
-1.  If {{source}} is null, does not currently have a [=CSS layout box=], or if
-    its layout box is not a [=scroll container=], return an unresolved time
-    value.
+The [=phase=] of a {{ScrollTimeline}} is calculated from the first matching
+condition from below:
+
+<div class="switch">
+
+:   If <em>any</em> of the following are true:
+
+    1.  {{source}} is null, or
+    1.  {{source}} does not currently have a [=CSS layout box=], or
+    1.  {{source}}'s layout box is not a [=scroll container=], or
+    1.  [=effective scroll range=] is null
 
-1.  Otherwise, let <var>current scroll offset</var> be the current scroll offset
+::  return [=timeline inactive phase|inactive=]
+
+:   Otherwise, let <var>current scroll offset</var> be the current scroll offset
```

Comment:
Something is amiss with the markup here. I'm not sure what it is supposed to look like but it currently renders like this:

![image](https://user-images.githubusercontent.com/1232595/84483811-d7b18100-acd4-11ea-8313-4dccf3dbd273.png)

Might I suggest the first step should be an ordered list with a nested unordered list? e.g.

1. If any of the following are true:
    * source is null, or
    * source does not currently have a CSS layout box, or
    * source's layout box is not a scroll container, or
    * effective scroll range is null

    return inactive and abort these steps.
2. Let current scroll offset be the current scroll offset of source in the direction specified by orientation.
3. Return the result corresponding to the first matching condition from below:
...

Reply:
I based the formatting on https://drafts.csswg.org/web-animations/#the-current-time-of-an-animation but I like your suggestion better. I'll update it in the next commit.